yb-voyager/cmd/exportData.go

```diff
@@ -152,6 +152,13 @@ func exportData() bool {
 		utils.ErrExit("Failed to connect to the source db: %s", err)
 	}
 	defer source.DB().Disconnect()
+	if changeStreamingIsEnabled(exportType) && bool(startClean) {
```

Comment:
See if you can repurpose `checkDataDirs` to also handle this. Would be better to have all startClean logic in one place. 
- Change the name of the func to `ClearMigrationStateIfRequired` 
- Call it in exportData after init of sourceDB instead of in `exportDataCommandFn`. (double-check that it is okay to do this, and won't cause other issues)

Reply:
Ironically, I'm running into a case with import-data where it doesn't seem possible to put all the startClean Logic in the same place. 
Lmk if you're running into anything similar here!